dcc-portal-ui/app/scripts/software/views/software.html

```diff
@@ -61,6 +61,43 @@ <h1 data-ui-scrollfix="80">
          </div>
       </section>
    </article>
+ <article>
+      <section id="icgc-get" class="tab-content" style="padding-top:0px;">
+         <h2>ICGC get</h2>
+         <div>
+            <p>To interact with all ICGC data repositories, you can use the ICGC get software. For system requirements and usage information, please see the <a href="http://docs.icgc.org/cloud/icgc-get/"><i class="icon-external-link"></i>User Guide</a>.</p>
+
+            <h3>Latest Release Zipfile</h3>
+            <p>Users are encouraged to use the latest version of our software to keep current with bug fixes, enhancements and new features. Click on the following to download the latest client version:</p>
+            <p>
+               <a id="latest-get-version" class="t_button" target="_blank" href="api/v1/ui/software/icgc-get/latest/osx">Download for Mac</a>
+               <a id="latest-get-version" class="t_button" target="_blank" href="api/v1/ui/software/icgc-get/latest/linux">Download for Linux</a>
+            </p>
+            <h3>Previous Release Zipfiles</h3>
+            <p>It is strongly recommended to use the latest release to take advantage of newest features and bug fixes.</p>
+
+            <p>However, if you wish to use an old version you may choose from the following:</p>
+            <table class="table table-striped">
+               <thead>
+                  <tr>
+                     <th>Version</th>
+                     <th>Mac</th>
+                     <th>Linux</th>
+                  </tr>
+               </thead>
+               <tbody id="get-versions">
+                  <tr data-ng-repeat="v in icgcGetVersions">
+                     <td>{{ v.uri }}</td>
+                     <td><a target="_blank" href="api/v1/ui/software/{{ icgcGet.artifactId }}{{ v.uri }}/osx">{{ icgcGet.artifactId }}-{{ v.uri }}-osx-dist.zip</a></td>
+                     <td><a target="_blank" href="api/v1/ui/software/{{ icgcGet.artifactId }}{{ v.uri }}/linux">{{ icgcGet.artifactId }}-{{ v.uri }}-linux-dist.zip.md5</a></td>
+                  </tr>
+               </tbody>
+            </table>
+            <h3>Docker Images</h3>
```

Comment:
I think this needs a bit more explaining since this is only for the dependencies right?